src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem/TodoItem';
+
+type Props = {
+  todos: Todo[];
+  deleteTodo: (id: number) => void;
+};
+
+export const TodoList: React.FC<Props> = ({ todos, deleteTodo }) => {
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots and `ul` paddings